plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -67,9 +80,31 @@ public void createControl(Composite parent) {
     projectIdField.setLayoutData(projectIdPosition);
     projectIdField.addModifyListener(pageValidator);
 
+    // Manage APIs
+    addManageLibrariesWidgets(container);
+
     Dialog.applyDialogFont(container);
   }
 
+  private void addManageLibrariesWidgets(Composite container) {
+    apiGroup = new Group(container, SWT.NONE);
+    apiGroup.setText("Manage Libraries");
+    GridDataFactory.fillDefaults().span(2, 1).applyTo(apiGroup);
+
+    for (Library library : getLibraries()) {
+      Button libraryButton = new Button(apiGroup, SWT.CHECK);
+      libraryButton.setText(library.getId());
+      libraries.put(libraryButton, library);
+    }
+
+    GridLayoutFactory.fillDefaults().applyTo(apiGroup);
+  }
+
+  // mock method until the libraries are defined via an extension point
```

Comment:
I think being able to configure them independently of an extension point is a good thing though.